backend/api/Controllers/Models/EchoInspection.cs

```diff
@@ -43,18 +47,23 @@ public class EchoInspectionComparer : IEqualityComparer<EchoInspection>
         public bool Equals(EchoInspection? e1, EchoInspection? e2)
         {
             if (ReferenceEquals(e1, e2))
+            {
                 return true;
+            }
 
             if (e2 is null || e1 is null)
+            {
                 return false;
+            }
 
             return e1.InspectionType == e2.InspectionType
-                && e1.TimeInSeconds == e2.TimeInSeconds;
+                   && e1.TimeInSeconds == e2.TimeInSeconds
+                   && e1.InspectionPoint == e2.InspectionPoint;
```

Comment:
Will this work even if the Position class does not override an Equals method?

Reply:
Good catch and I'm not sure. Decided to override the equals method the same way as has been done for orientation.